client/src/stylesheets/components/_collapsePanel.scss

```diff
@@ -9,9 +9,9 @@
 
   .collpase-panel-title {
```

Comment:
typo: .collpase-panel-title -> .collapse-panel-title

Reply:
this scss typeo class exists in CollapsePanel.js too.  can you swing by and fit it real fast too? Thanks

Reply:
fixed in all places this typo has occured